lib/python/tests/test_model.py

```diff
@@ -53,6 +53,14 @@ def test_create_get_from_id_and_update(
     assert model.model_id == get_model.model_id
 
 
+@pytest.mark.integration
+def test_search_models(integration_client):
+    models = Model.search(client=integration_client)
+
+    for model in models:
+        assert isinstance(model, Model)
```

Comment:
As well as this could we test for failure and some edge cases?

Eg.
- Test a variety of each of arguments.
- Assert you can retrieve a specific model

Reply:
Added an additional test to look for specific models

Reply:
Could we test with different arrangements of task, libraries and filters?